src/OrchardCore.Modules/OrchardCore.AuditTrail/Views/AuditTrailEvent-Content.SummaryAdmin.cshtml

```diff
@@ -0,0 +1,59 @@
+﻿@using OrchardCore.ContentManagement
+@using OrchardCore.AuditTrail.Providers
+@using OrchardCore.AuditTrail.Extensions
+
+@{
+    var contentItem = (ContentItem)Model.ContentItem;
+    var auditTrailEvent = (AuditTrailEvent)Model.AuditTrailEvent;
+    var eventDescriptor = (AuditTrailEventDescriptor)Model.Descriptor;
+    var versionNumber = (int)Model.VersionNumber;
+    var versionText = T[$"Version {versionNumber}"];
```

Comment:
You can't localize dynamic strings.

Reply:
`T["Version {0}", versionNumber"]`

Reply:
If I use `var versionText = T["Version {0}", versionNumber]`, the value of the `versionText` will be Version {0}. For some reason, it will not resolve the {0} placeholder with the value of the versionNumber variable. But it works if I have the following line:
`var versionText = T[string.Format("Version {0}", versionNumber)];` It's strange.

Reply:
What about `versionNumber.ToString()`? Shouldn't be necessary though.

Reply:
I know, but that doesn't work either.

Reply:
I used `@T["Text {0}.", value]` this morning and it worked fine